setup.py

```diff
@@ -41,21 +41,23 @@
 ]
 
 install_requires = [
-    'adage==0.8.5',
+    'adage==0.10.0',
     'celery>=4.1.0,<4.3',
     'click>=7',
     'enum34>=1.1.6',
-    'packtivity==0.10.0',
-    'pyOpenSSL==17.5.0',  # FIXME remove once yadage-schemas solves deps.
+    'jq==0.1.6',
+    'networkx==2.3',
+    'packtivity==0.14.21',
+    # 'pyOpenSSL==17.5.0',  # FIXME remove once yadage-schemas solves deps.
     'reana-commons>=0.6.0.dev20190619,<0.7.0',
-    'requests==2.20.0',
-    'rfc3987==1.3.7',  # FIXME remove once yadage-schemas solves deps.
-    'strict-rfc3339==0.7',  # FIXME remove once yadage-schemas solves deps.
+    'requests==2.22.0',
+    # 'rfc3987==1.3.7',  # FIXME remove once yadage-schemas solves deps.
+    # 'strict-rfc3339==0.7',  # FIXME remove once yadage-schemas solves deps.
     'SQLAlchemy-Utils>=0.32.18',
     'SQLAlchemy>=1.1.14',
-    'yadage-schemas==0.7.16',
-    'yadage==0.13.5',
-    'webcolors==1.7',  # FIXME remove once yadage-schemas solves deps.
+    'yadage-schemas==0.10.6',
+    'yadage==0.20.0',
+    # 'webcolors==1.7',  # FIXME remove once yadage-schemas solves deps.
```

Comment:
Please check also `reana-client`, we have `yadage-schemas` and friends there, used for workflow validation.  So we need to update both concurrently.

Reply:
(Ditto for `reana-workflow-monitor` component, you could in theory update it there, although we don't really use it anymore, so we can left it alone... or we could add a deprecation commit there.  Mentioning it here just for the sake of completeness.)